.github/workflows/metrics.yml

```diff
@@ -65,4 +65,5 @@ jobs:
 
         env:
           SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+          TZ: America/Chicago
```

Comment:
I think you need to put the env in [these two steps](https://github.com/path-cc/metric-tools/pull/18/files#diff-42d31188c1d75b9460b01dc33518bf940a893261f8ea207e3cd6a1d5d485e020R19-R54) instead

Reply:
Ah, good catch, i got the yaml hierarchy wrong.  (I was thinking that env applied to all steps.)